eelbrain/_trf/tests/test_boosting.py

```diff
@@ -224,3 +225,56 @@ def test_boosting_func():
     assert rr == approx(mat['crlt'][1, 0])
     # svdboostV4pred multiplies error by number of predictors
     assert_allclose(test_sse_history, mat['Str_testE'][0] / 3)
+
+@pytest.mark.parametrize('n_workers', [0, True])
+def test_trf_len(n_workers):
+    configure(n_workers=n_workers)
+    # test vanilla boosting
+    x = NDVar(np.random.normal(0, 1, 1000), UTS(0, 0.1, 1000),name='x')
```

Comment:
I would use a seeded random state to avoid random surprises in test runs
```suggestion
    rng = np.random.RandomState(0)
    x = NDVar(rng.normal(0, 1, 1000), UTS(0, 0.1, 1000), name='x')
```
...

Reply:
added seed